docs/branch_based_planner/getting-started.md

```diff
@@ -0,0 +1,80 @@
+# Getting Started With Branch Planner
+
+If the Branch Planner is enabled through Helm values, the Branch Planner will
+watch all configured Terraform resources, check their referenced Source, and
+polls for Pull Requests using GitHub's API with the provided token.
+
+When an open Pull Request is detected, the Branch Planner creates a new or
+updates an existing Terraform object with Plan Only mode from the original
+Terraform object.
+
```

Comment:
```suggestion
When the Branch Planner detects an open Pull Request, it either creates a new Terraform object or
updates an existing one, applying Plan Only mode from the original
Terraform object.

```

Reply:
I had a little trouble parsing this part but hopefully interpreted correctly?

Reply:
This version sounds better, but as I read I would assume it applies the Plan Only mode on the original Terraform object. It creates a clone of the original object and applies plan only mode on it, but it does not change anything on the original object.

Maybe "applying Plan Only mode based on the original Terraform object", idk :/

Reply:
"based on" +1